doc/source/index.rst

```diff
@@ -255,6 +255,131 @@ the same start and end times and coverage, just different data.
+        more wanted files are missing, a message will be published without
+        the missing files.  If all wanted files
+        are present before timeout is reached, collection is finished and a
+        message will be published immediately.
+        The syntax is as for ``critical_files``.
+
+    all_files
+        Describes files that are accepted, but not needed.  Any file matching the
+        ``all_files`` pattern is included with the published message, but the
+        segment gatherer will not wait for those files.
+
+    is_critical_set
+        ???
+
+    variable_tags
+        List of strings for tags that are expected to vary between segments.
+        Those are replaced with wildcards for the purposes of pattern matching.
+
+    group_by_minutes
+        Optional.
+        Group the data for every full minute interval
+        For example in the case time of "201712081129" would go in slot
+        "2017-12-08T11:20:00" along with other files with minutes between 20 and 29.
+        (Can also be defined globally)
+        By default, no grouping by minutes is performed and times are matched
+        exactly or with a tolerance of ``time_tolerance``.
+
+    start_time_pattern
+        Optional.
+        Mapping with the keys ``start_time``, ``end_time``, and ``delta_time``,
+        which are all strings with the format ``%H:%M``.  This defines a pattern
+        of time slots that will be considered for processing.  Any timeslot that
+        does not match this pattern will be discarded.  For example, a
+        ``start_time`` of ``06:00``, ``end_time`` of ``18:00``, and ``delta_time``
+        of ``01:00`` will result in processing to go ahead only for whole-hour
+        time slots between 06:00 and 18:00.
+        By default, all time slots are processed.
+
+    keep_parsed_keys
+        Optional.
+        The segment gatherer normally combines metadata from the filename and
+        the received posttroll message.  The list of keys defined here will be
+        taken from the filename pattern rather than from the message metadata.
+        By default, only the parsed keys hardcoded in the source code are always
+        taken from the filename pattern.
+        (Can also be defined globally)
+
+timeliness
+    Time in seconds from the first arrived file until timeout.  When timeout is
+    reached, all collected files (meaning all files that match the ``all_files`` pattern)
+    are broadcast in a posttroll message.
+
+time_name
+    Name of the time tag used in all patterns.
+
+time_tolerance
+    Time difference in seconds for which start times are considered to
+    belong to the same time slot.
+
+posttroll
+    Configuration related to posttroll messaging, with the keys ``topics`` (list of topics to listen to)
+    ``publish_topic`` (topic used for published messages), ``publish_port``, ``nameservers``, and ``addresses``.
+
+bundle_datasets
+    Optional.
+    ???
```

Comment:
I guess this is merging `dataset`s within a `collection` to a single `dataset`. Not sure, though.

Reply:
Improved formulation in 0f099f8.